README.md

```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
```

Comment:
Wonderful.
Let's just get rid of some lines that should be deleted when resolving conflicts like this one.
The lines that contains <<<<<< >>>>>>> ====== and the name of branches should be removed.